trigger.go

```diff
@@ -63,45 +110,67 @@ func (api *API) TriggersCreate(triggers Triggers) (err error) {
 	result := response.Result.(map[string]interface{})
 	triggerids := result["triggerids"].([]interface{})
 	for i, id := range triggerids {
-		triggers[i].TriggerId = id.(string)
+		triggers[i].TriggerID = id.(string)
 	}
 	return
 }
 
-// Wrapper for item.delete: https://www.zabbix.com/documentation/2.2/manual/appendix/api/item/delete
-// Cleans ItemId in all items elements if call succeed.
+// TriggersUpdate Wrapper for trigger.update
+// https://www.zabbix.com/documentation/3.2/manual/api/reference/trigger/update
+func (api *API) TriggersUpdate(triggers Triggers) (err error) {
+	_, err = api.CallWithError("trigger.update", triggers)
+	return
+}
+
+// TriggersDelete Wrapper for trigger.delete
+// Cleans ItemId in all triggers elements if call succeed.
+// https://www.zabbix.com/documentation/3.2/manual/api/reference/trigger/delete
 func (api *API) TriggersDelete(triggers Triggers) (err error) {
 	ids := make([]string, len(triggers))
 	for i, trigger := range triggers {
-		ids[i] = trigger.TriggerId
+		ids[i] = trigger.TriggerID
 	}
 
 	err = api.TriggersDeleteByIds(ids)
 	if err == nil {
 		for i := range triggers {
-			triggers[i].TriggerId = ""
+			triggers[i].TriggerID = ""
 		}
 	}
 	return
 }
 
-// Wrapper for item.delete: https://www.zabbix.com/documentation/2.2/manual/appendix/api/item/delete
+// TriggersDeleteByIds Wrapper for trigger.delete
+// https://www.zabbix.com/documentation/3.2/manual/api/reference/trigger/delete
 func (api *API) TriggersDeleteByIds(ids []string) (err error) {
+	deleteIds, err := api.TriggersDeleteID(ids)
```

Comment:
Why have both `TriggersDeleteByIds()` and `TriggersDeleteID()` functions?

As `TriggersDeleteID()` also accepts a list of ids, it should be named `TriggersDeleteIDs()` if really needed.